File: lc81.java
public class lc81 {
    // 81. Search in Rotated Sorted Array II
    public static void main(String[] args) {
        int nums[]={2,5,6,0,0,1,2};
        int target=0;
        boolean ans=false;
        for (int i=0;i< nums.length;i++){
            if (nums[i]==target){
                ans=true;
                break;
            }
        }
        System.out.println(ans);
    }
}

File: lc81.py
from typing import List


class Solution:
    def search(self, nums: List[int], target: int) -> bool:
        ans=False
        nums.sort()
        for i in nums:
            if i==target:
                ans=True
                break
            elif i>target:
                break
        return ans
print(Solution().search(nums=[1,2,3,4,5,6,7], target=10))